bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/service/StateDescriptionService.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.core.service;
+
+import java.util.Locale;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.types.StateDescription;
+
+/**
+ * Implementations of this service provide strategies for merging info from
+ * different StateDescriptionProviders into one StateDescription.
+ * 
+ * @author Lyubomir Papazov - Initial contribution
+ *
+ */
+@NonNullByDefault
+public interface StateDescriptionService {
+
+    /**
+     * Implementations of this method merge the StateDescriptions provided from
+     * multiple StateDescriptionProviders into one final StateDescription
+     * 
+     * @param itemName
+     * 
```

Comment:
Please sue the default ESH formatter here once to get rid of the obsolete empty lines.